content/en/logs/log_collection/kotlin_multiplatform.md

```diff
@@ -84,163 +84,24 @@ kotlin {
-   ```kotlin
-    // in common source set
-    fun initializeDatadog(context: Any? = null) {
-        // context should be application context on Android and can be null on iOS
-        val appClientToken = <CLIENT_TOKEN>
-        val appEnvironment = <ENV_NAME>
-        val appVariantName = <APP_VARIANT_NAME>
-
-        val configuration = Configuration.Builder(
-                clientToken = appClientToken,
-                env = appEnvironment,
-                variant = appVariantName
-        )
-            .useSite(DatadogSite.US5)
-            .trackCrashes(true)
-            .build()
-
-        Datadog.initialize(context, configuration, trackingConsent)
-    }
-   ```
-   {{< /site-region >}}
-
-   {{< site-region region="gov" >}}
-   ```kotlin
-    // in common source set
-    fun initializeDatadog(context: Any? = null) {
-        // context should be application context on Android and can be null on iOS
-        val appClientToken = <CLIENT_TOKEN>
-        val appEnvironment = <ENV_NAME>
-        val appVariantName = <APP_VARIANT_NAME>
-
-        val configuration = Configuration.Builder(
-                clientToken = appClientToken,
-                env = appEnvironment,
-                variant = appVariantName
-        )
-            .useSite(DatadogSite.US1_FED)
-            .trackCrashes(true)
-            .build()
-
-        Datadog.initialize(context, configuration, trackingConsent)
-    }
-   ```
-   {{< /site-region >}}
-
-   {{< site-region region="ap1" >}}
-   ```kotlin
-    // in common source set
-    fun initializeDatadog(context: Any? = null) {
-        // context should be application context on Android and can be null on iOS
-        val appClientToken = <CLIENT_TOKEN>
-        val appEnvironment = <ENV_NAME>
-        val appVariantName = <APP_VARIANT_NAME>
-
-        val configuration = Configuration.Builder(
-                clientToken = appClientToken,
-                env = appEnvironment,
-                variant = appVariantName
-        )
-            .useSite(DatadogSite.AP1)
-            .trackCrashes(true)
-            .build()
-
-        Datadog.initialize(context, configuration, trackingConsent)
-    }
-   ```
-   {{< /site-region >}}
-   {{< site-region region="ap2" >}}
-   ```kotlin
-    // in common source set
-    fun initializeDatadog(context: Any? = null) {
-        // context should be application context on Android and can be null on iOS
-        val appClientToken = <CLIENT_TOKEN>
-        val appEnvironment = <ENV_NAME>
-        val appVariantName = <APP_VARIANT_NAME>
-
-        val configuration = Configuration.Builder(
-                clientToken = appClientToken,
-                env = appEnvironment,
-                variant = appVariantName
-        )
-            .useSite(DatadogSite.AP2)
-            .trackCrashes(true)
-            .build()
-
-        Datadog.initialize(context, configuration, trackingConsent)
-    }
-   ```
-   {{< /site-region >}}
+```kotlin
```

Comment:
I wasn't sure if it's a big deal, but in the original formatting, everything is aligned. The refactor isn't
```
.useSite(DatadogSite.EU1)
.trackCrashes(true)
.build()
```
vs.
```      
     .useSite(DatadogSite.EU1)
     .trackCrashes(true)
.build()
```